doc/source/index.rst

```diff
@@ -24,5 +24,8 @@ Link to gallery.
    :caption: User guide
 
    data_structures
+   topology_system
+   coordinate_transformation
```

Comment:
I wouldn't call it "coordinate_transformation" because it gets really confusing with the "trajectory transformations" and the "mathematical transformations". 

I understand that we are transforming coordinates into something else but perhaps adapt the title "working with coordinates" for the filename, too?